apps/cookbook/src/app/showcase/toggle-button-showcase/toggle-button-showcase.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="example">
+  <cookbook-toggle-button-example></cookbook-toggle-button-example>
+  <cookbook-code-viewer [html]="exampleHtml"></cookbook-code-viewer>
```

Comment:
We now embed the example inside `cookbook-example-viewer` , e.g. see `apps\cookbook\src\app\showcase\segmented-control-showcase\segmented-control-showcase.component.html`